src/App.js

```diff
@@ -1,10 +1,103 @@
 import React from 'react';
 import './App.css';
+import Counter from "./Components/Counter";
 
 function App() {
+    const[aardbeiCounter, setAardbeiCounter]= React.useState(0);
+    const[banaanCounter, setBanaanCounter]= React.useState(0);
+    const[appelCounter, setAppelCounter]= React.useState(0);
+    const[kiwiCounter, setKiwiCounter]= React.useState(0);
+
+    function reset (){
+        setAardbeiCounter(0)
+        setBanaanCounter(0)
+        setAppelCounter(0)
+        setKiwiCounter(0)
+    }
+
   return (
     <>
       <h1>Fruitmand bezorgservice</h1>
+
+        <div className="card">
+            🍓 Aardbeien
+           <Counter fruitName={aardbeiCounter} setFruitName={setAardbeiCounter}/>
+        </div>
```

Comment:
Ik de div en de tekst opnemen in het Counter component, want ik zie dat bij elk fruitsoort terugkomen en kun je daarom mooi hergebruiken